Distribution/Data/Crafting/alchemyplus.json

```diff
@@ -0,0 +1,396 @@
+{
+  "MainSkill": "Alchemy",
+  "GumpTitleId": 1044001,
+  "MinCraftDelays": 4,
+  "MaxCraftDelays": 4,
+  "Delay": 1.0,
+  "MinCraftChance": 0.0,
+  "CraftWorkSound": 550,
+  "CraftEndSound": 549,
+  "CraftEntries": [
+    {
+      "ItemType": "Server.Items.PhandelsFineIntellectPotion",
+      "Name": "Phandel's Fine Intellect",
+      "GroupName": 3000112,
+      "MinSkill": 30,
+      "MaxSkill": 30,
+      "Resources": [
+        {
+          "ItemType": "Server.Items.FertileDirt",
+          "Name": 1023969,
+          "Amount": 1,
+          "Message": "You do not have enough Fertile Dirt to make that."
```

Comment:
Mmm do we have to specify this whole string every time? Seems a bit wasteful to have the extra text when it's always the same